packages/shared/src/types/abTests/epic.ts

```diff
@@ -67,56 +67,50 @@ export interface EpicVariant extends Variant {
 
 export type ContributionFrequency = 'ONE_OFF' | 'MONTHLY' | 'ANNUAL';
 
-export interface AmountSelection {
+export const contributionTabFrequencies: ContributionFrequency[] = ['ONE_OFF', 'MONTHLY', 'ANNUAL'];
+
+interface ContributionTypeItem {
+    label: string;
+    suffix: string;
+}
+export type ContributionType = {
+    [key in ContributionFrequency]: ContributionTypeItem;
+};
+
+interface AmountValuesObject {
     amounts: number[];
     defaultAmount: number;
     hideChooseYourAmount?: boolean;
 }
 
-export type ContributionAmounts = {
-    [key in ContributionFrequency]: AmountSelection;
+export type AmountsCardData = {
+    [key in ContributionFrequency]: AmountValuesObject;
 };
 
-export interface AmountsTestVariant {
-    name: string;
-    amounts: ContributionAmounts;
+export interface AmountsVariant {
+    variantName: string;
+    defaultContributionType: ContributionFrequency;
+    displayContributionType: ContributionFrequency[];
+    amountsCardData: AmountsCardData;
+}
+
+export interface SelectedAmountsVariant extends AmountsVariant {
+    testName: string;
 }
 
 export interface AmountsTest {
-    name: string;
+    testName: string;
+    liveTestName?: string;
+    testLabel?: string;
     isLive: boolean;
-    variants: AmountsTestVariant[];
+    region: CountryGroupId | '';
+    country: string[];
```

Comment:
I still think [this thread](https://github.com/guardian/support-admin-console/pull/439#discussion_r1272214566) from the earlier PR is relevant. The model can make it clear that it's either a `CountryGroupId` or a `string[]`.
Happy to help implement this, including the scala encoder etc

Reply:
here's a proposal on the new SAC PR - https://github.com/guardian/support-admin-console/pull/477/files#r1300208356